pkg/controllers/disruption/consolidation_test.go

```diff
@@ -3583,4 +3584,241 @@ var _ = Describe("Consolidation", func() {
 			Expect(result.RequeueAfter).To(BeNumerically(">", 0))
 		})
 	})
+	Context("Metrics", func() {
+		var nodeClaims []*v1beta1.NodeClaim
+		var nodes []*v1.Node
+		var rs *appsv1.ReplicaSet
+		var numNodes int
+		var labels map[string]string
+
+		BeforeEach(func() {
+			numNodes = 25
+			labels = map[string]string{
+				"app": "test",
+			}
+			nodeClaims, nodes = test.NodeClaimsAndNodes(numNodes, v1beta1.NodeClaim{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{
+						v1beta1.NodePoolLabelKey:     nodePool.Name,
+						v1.LabelInstanceTypeStable:   leastExpensiveInstance.Name,
+						v1beta1.CapacityTypeLabelKey: leastExpensiveOffering.CapacityType,
+						v1.LabelTopologyZone:         leastExpensiveOffering.Zone,
+					},
+				},
+				Status: v1beta1.NodeClaimStatus{
+					Allocatable: map[v1.ResourceName]resource.Quantity{
+						v1.ResourceCPU:  resource.MustParse("32"),
+						v1.ResourcePods: resource.MustParse("100"),
+					},
+				}},
+			)
+			// create our RS so we can link a pod to it
+			rs = test.ReplicaSet()
+			ExpectApplied(ctx, env.Client, rs)
+		})
+		It("should update metrics for empty-nodeclaim consolidation", func() {
```

Comment:
Should we create a `Context()` for "eligible" metrics since we have multiple tests that cover these metrics?